web/src/integration_url_modal.ts

```diff
@@ -27,7 +27,7 @@ export function show_generate_integration_url_modal(api_key: string): void {
     };
     const html_body = render_generate_integration_url_modal({
         default_url_message,
-        max_topic_length: realm.max_topic_length,
+        max_topic_length: realm.max_message_length,
```

Comment:
Why have you changed this?

Reply:
@afeefuddin Yeah, that was necessary. Can you check recent update to the `state_data.ts`

Reply:
Yeah, I made that update in `state_data.ts`, but why are you introducing logic bugs? How did you arrive at the conclusion that `max_topic_length` and `max_message_length` are the same fields?